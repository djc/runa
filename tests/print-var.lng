# test: {"args": ["cold"]}
def __main__(name: str, args: list[str]) -> int:
	x = 29
	print(str(x))
	y = 'birthday'
	print(y)
	print(name)
	print(args[0])
