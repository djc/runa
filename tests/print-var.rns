# test: {"args": ["cold"]}
def main(name: &Str, args: &Array[Str]):
	x = 29
	print(Str(x))
	y = 'birthday'
	print(y)
	print(name)
	print(args[0])
