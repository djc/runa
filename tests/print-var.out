29
birthday
tests/print-var.test
cold
